scripts/reclaim-space.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env sh
+
+#
+# The script tries to free as much space as possible by removing nix packages
+# and docker images that aren't used.
+#
+
+set -e
+
+MIN_FREE_GB=$1
+
+get_avail_gb() {
+    echo $(( $(df --output=avail / | awk 'NR == 2 { print $1 }' ) / 1024 / 1024 ))
```

Comment:
```suggestion
    echo $(( $(df --output=avail / | awk 'NR == 2 { print $1 }' ) / 1000 / 1000 ))
```

You are using `GB` throughout the script. So either this or change `GB` to `GiB` everywhere (ideally in the name of the method as well).

Reply:
Please use GiB and 1024.